lib/sequelize.js

```diff
@@ -199,6 +199,8 @@ module.exports = (function() {
     this.importCache = {};
 
     Sequelize.runHooks('afterInit', this);
+
+    Promise.prototype.sequelize = this;
```

Comment:
What about this:

```js
var Promise = require( 'lib/promise' )
// ...

function Sequelize () {
   // ...
  this.Promise = function () {
    return Promise.apply( this, arguments );
  };
  this.Promise.prototype = Object.create( Promise.prototype );

  return this;
}

```

And on every sequelize promise return:
```
  new sequelize.Promise();
```

Does it feel ugly?